tests/parallel/quasi-newton/helpers.cpp

```diff
@@ -60,13 +60,13 @@ void runTestQN(std::string const &config, TestContext const &context)
     interface.readBlockScalarData(readDataID, 4, vertexIDs, inValues);
 
     /*
-      Solves the following non-linear equations, which are extended to a fixed-point equation (simply +x)
+      Solves the following non-linear equations, which are extended to a fixed-point equation (simply +x_1)
```

Comment:
No, "+x" was correct. You need to add the complete input vector. Otherwise, you don't get a fixed-point equation. See lines 78 to 81 below.

Reply:
Fixed. 62fae13a431b10e77d0201e2510184efc4fe7f5c